internal/kafka/internal/api/dbapi/kafka_request_types.go

```diff
@@ -261,3 +263,46 @@ func (k *KafkaRequest) IsUsingSharedTLSCertificate(kafkaConfig *config.KafkaConf
 func (k *KafkaRequest) IsADeveloperInstance() bool {
 	return k.InstanceType == types.DEVELOPER.String()
 }
+
+//////////////////////////////////
+/// functional builder
+
```

Comment:
This seems to have been done for testing purposes.

Could we scope the names and the file where it belongs more to be more concrete?